src/mcts/search.cc

```diff
@@ -130,12 +130,13 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
 
   int multipv = 0;
   const auto default_q = -root_node_->GetWL();
```

Comment:
This should be default_wl, and there probably should be a separately calculated default_q which takes into account drawscore correctly (since you an't just negate the parent GetQ) - let me know if you want it fixed in this PR.

Reply:
okay you can just negate parent GetQ (if you negate drawscore at the same time...) - since I made this sound harder than it actually is.

Reply:
Implemented now.